src/pages/index.jsx

```diff
@@ -57,7 +57,7 @@ function EpisodeEntry({ episode }) {
             className="order-first font-mono text-sm leading-7 text-slate-500"
           />
           <p className="mt-1 text-base leading-7 text-slate-700">
-            {episode.description}
+            {episode.description.replace(/<\/?[^>]+(>|$)/g, "")}
           </p>
```

Comment:
Since we are receiving rich text that can contain HTML tags, maybe you can consider injecting the description using `dangerouslySetInnerHTML`. Don't forget to sanitize the innerHTML. 

[Reference](https://blog.logrocket.com/using-dangerouslysetinnerhtml-in-a-react-application/) 
```suggestion 
<p className="mt-1 text-base leading-7 text-slate-700" 
dangerouslySetInnerHTML={{__html: episode.description}}
/>
```

Reply:
I suggest something like https://github.com/cure53/DOMPurify to sanitize the input